tock/tock/templates/_navigation.html

```diff
@@ -24,6 +24,11 @@
           <span>Users</span>
         </a>
       </li>
+      <li>
+        <a href="{% url 'logout' %}" class="usa-nav-link">
+          <span>Log out</span>
+        </a>
```

Comment:
Normally we'd guard this with a check to see if the current user is actually logged in, but since that's guaranteed (except for the `logout` view and error views, which this partial isn't included in), I didn't do that.